services/ui-src/src/forms/mcpar/index.ts

```diff
@@ -3,15 +3,19 @@ import reportSchema from "./mcpar.schema";
 // utils
 import {
   addValidationToReportJson,
+  copyAdminDisabledStatusToForms,
   flattenReportRoutesArray,
 } from "utils/reports/reports";
 import { ReportJson } from "types";
 
+const stateOrAdminVersion: ReportJson =
+  copyAdminDisabledStatusToForms(rawReportJson);
```

Comment:
Nit: Would maybe name this something like `reportJsonWithDisabledStatus` instead.
```suggestion
const reportJsonWithDisabledStatus: ReportJson =
  copyAdminDisabledStatusToForms(rawReportJson);
```